chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -44,6 +44,9 @@ def find_node_boot_mac_addresses(node, admin_data_net)
       next if addr_data["family"] != "lladdr"
       result << addr unless result.include? addr
     end
+    # add permanent hardware addresses, that may be hidden for slave interfaces of a bond
+    permanent_addr = node["crowbar_ohai"]["detected"]["network"][interface]["addr"] rescue nil
```

Comment:
I'm looking into my env at that `crowbar_ohai` data and doesn't seem to include that addr field:

```
crowbar_ohai:
  detected:
    network:
      eth0:
        path:    0000:00/0000:00:03.0/0000:01:01.0/virtio4
        speeds:
            1g
            0g
      eth1:
        path:    0000:00/0000:00:03.0/0000:01:02.0/virtio5
        speeds:
            1g
            0g
```

what am I missing?

Reply:
I cannot find it neither on another environment with two bonds (eth0+eth3 and eth1+eth2):

```
crowbar_ohai:
  detected:
    network:
      eth0:
        path:    0000:00/0000:00:02.2/0000:04:00.0
        speeds:
            1g
            10g
      eth1:
        path:    0000:00/0000:00:02.2/0000:04:00.1
        speeds:
            1g
            10g
      eth2:
        path:    0000:00/0000:00:02.0/0000:05:00.0
        speeds:
            1g
            10g
      eth3:
        path:    0000:00/0000:00:02.0/0000:05:00.1
        speeds:
            1g
            10g
```

Reply:
@abel-navarro It's added by the first commit in this pull request :-)

Reply:
@vuntz ah, i missed that part. Thanks.